src/app/src/main/kotlin/org/vaccineimpact/api/app/context/RequestBodySource.kt

```diff
@@ -17,5 +18,11 @@ sealed class RequestBodySource
         override fun getContent(context: ActionContext)
                 = context.getPart(partName)
     }
+
+    class InMemory(private val contents: String) : RequestBodySource()
+    {
```

Comment:
Given that the one method this class overrides has an unused method argument, I feel like this isn't the right construction and somewhat obfuscates what's really going on. See comment below on the `ModellingGroupController` for an alternative suggestion.